### ISSUE TITLE
[checked_identities] - URL creation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#635]
```
Token: 884e80e99ca141cce3ddce8731218483 (Don't modify this token!)
Version: 2.8.11 (30-11-2013)
PHP: 5.4.17
RCMAIL: 0.9.5
Database: mysql
SERVER: Apache/2.2.3 (CentOS)
----
I.  Issue Description:

II. Steps to reproduce the Issue:
1.Creating an new Identity
2.Wrong URL with a wrong Servername (Clustername)

We are using a Cluster of several Apache-Server and the enviromentals HTTPS, 
HTTP_HOST, SERVER_PORT cannot be used here to create the URL automatic. It 
would be better to be able to type in the "basis uri" in the config as an 
option.

old checked_identities.php:
$AB="http$t://".$_SERVER['HTTP_HOST'].':'.$_SERVER['SERVER_PORT'].$_SERVER['PHP_
SELF'];

new checked_identities.php:
$AB=($A->config->get('basis_uri')?:"http$t://".$_SERVER['HTTP_HOST'].':'.$_SERVE
R['SERVER_PORT']).$_SERVER['PHP_SELF']

And adding the following line in the config:
$rcmail_config['basis_uri'] = "https://<OUR SERVERNAME>";

```

Original issue reported on code.google.com by `szrzs...@gmail.com` on 7 Feb 2014 at 12:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `myroundc...@gmail.com` on 16 Feb 2014 at 2:39
- Changed title: **[checked_identities] - URL creation**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Roundcube 1.0 branch.
```

Original comment by `roland.l...@gmail.com` on 21 Feb 2014 at 8:24
- Changed state: **Fixed**
